kolibri/core/webpack/hooks.py

```diff
@@ -272,6 +277,16 @@ def frontend_message_tag(self):
         else:
             return []
 
+    def context_tag(self):
+        if self.context:
+            return [
+                '<script>window.{bundle} = JSON.parse("{context}");</script>'.format(
```

Comment:
It might be preferable to keep all the context objects namespaced together. e.g. somewhere at the top 

```js
window.kolibriPluginContextObjects = {};
```

and then for each `context_tag`:

```js
window.kolibriPluginContextObjects['{bundle}'] = JSON.parse('{context}');
```

to prevent plugins from clobbering the global namespace, or potentially using names that aren't valid JS identifiers.

ref: https://github.com/learningequality/kolibri-sentry-plugin/blob/master/kolibri_sentry_plugin/assets/src/module.js#L10

Reply:
Yeah, that sounds like a good idea. I was also thinking about adding a `getContext` helper function that would mean the plugin itself wouldn't have to worry about the implementation detail of where the context was stored, sound worthwhile?

Reply:
>  I was also thinking about adding a getContext helper function

yes, that's a good idea

Reply:
i wonder if 'context' is the right word... is this more 'options'?

Reply:
so, presumably we could use this new functionality here, right?

https://github.com/learningequality/kolibri/pull/5847#discussion_r313025261

Reply:
Yes - and we can also use it to consolidate all of our `default_frontend` global assignment that currently happens in a bunch of different template tags.